src/ontology/analysis/analyze_mondo_stats/README.md

```diff
@@ -0,0 +1,307 @@
+## Overall Mondo Stats
+
+### Description
+This notebook and SPARQL queries are to generate data as requested [here](https://docs.google.com/document/d/1vPl2x8gfTypjzcb9ND8qXXfKIjPDE4RwuExmzAUpM6Q/edit?tab=t.0).
+
+
+### Request 1: Alignment with the source - Total number of diseases in sources
+For Mondo terms are in the human disease branch, i.e. children of MONDO:0700096 'human disease', get the total number of diseases represented by each source of interest (see below). This should be the count of distinct Mondo classes that contain at least one database_cross_reference to each of the "Sources of Interest". The query can be run against `mondo-edit.obo`, but must use the parameter `reason` with the robot command since many Mondo classes only classify with a parent of MONDO:0700096 'human disease' in the inferred hierarchy.
+
+NOTE: A list of all NCIT terms in the neoplasm branch is needed in order to filter all Mondo xrefs to NCIT where these NCIT terms are in the neoplasm branch since Mondo contains NCIT xrefs to terms both in the neoplasm branch and other NCIT branches.
+
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_xref.ru data/output/all_human_disease_mondo_class_xref_count.tsv
+```
+
+
+#### Sources of Interest:
+- DOID
+- NCIT (neoplasm branch) --> special processing needed
+- OMIM
+- Orphanet
+- ICD10CM
+- UMLS (as provided by MedGen)
```

Comment:
How do we count the UMLS? 
Counting how many are given to MedGen is not representative of how many are in the source itself.

Reply:
It's where the xref value is from UMLS. I don't recall why a mention of MedGen is here from my notes when I talked to Nico about this in our 1:1.